src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin square ($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin square-style {
+  border: 1px solid #000;
+  background-color: #eee;
+}
+
+@mixin flex-item-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin hover-transform-style {
+  background-color: pink;
+  transform: translateY(-20px);
+  transition: background-color 0.5s,
```

Comment:
remove this property from `:hover` to the class for it to work it both directions(mouse-over and mouse-leave)